public/scripts/report.js

```diff
@@ -1,6 +1,6 @@
 (function() {
   // add listener to get location button
-  var locationButton = document.getElementById('getLocationButton');
+  var locationButton = document.getElementById('location');
```

Comment:
Actually I think this id does need to stay as getLocation because there are two types of location submission: 

getlocation (using users current location)
and
searchLocation (where the user selects a location from the map)

So 'location' will be too vague